start.sh

```diff
@@ -85,6 +86,7 @@ prepare_configs
 [ -n "${REV_SERVER_DOMAIN}" ] && change_setting "REV_SERVER_DOMAIN" "$REV_SERVER_DOMAIN"
 [ -n "${REV_SERVER_TARGET}" ] && change_setting "REV_SERVER_TARGET" "$REV_SERVER_TARGET"
 [ -n "${REV_SERVER_CIDR}" ] && change_setting "REV_SERVER_CIDR" "$REV_SERVER_CIDR"
+[ -n "${RATE_LIMIT}" ] && change_setting "RATE_LIMIT" "$RATE_LIMIT"
```

Comment:
This wont set the FTL config file, rather `setupVars.conf` (which is slowly being phased out). Take a look at lines 80 and 81 (`${ServerIP}` and `${ServerIPv6}`)

Reply:
Oops, I new that but made a copy-paste error. Fixing now!